### ISSUE TITLE
Error Code: 2168-0002 (0x4a8) 

### ISSUE THREAD

**Issue by NaturalHarmonia** [WerWolv/Tesla-Menu#41]
I got that error code with Program: 4200000000007E51A and Firmware: 13.1.0 (Atmosphere 1.2.1-master0fb59d0ad4)

I get that often when using cheats in Brilliant Diamond and Shining Pearl. I don't know what the problem is. If I delete that folder I can't use Tesla Menu...

**Comment by masagrator** [comment]
Update atmosphere and reduce your custom sysmodules from contents. Probably you have too much installed.

**Comment by NaturalHarmonia** [comment]
What exactly is a sysmodule? Those folder with all those numbers under the contents directory?

**Comment by eqixcx** [comment]
> Update atmosphere and reduce your custom sysmodules from contents. Probably you have too much installed.

I don't think this is the issue, I'm running latest atmosphere, and if I boot with *only* Tesla and sys-clk, I crash on boot with the same error, Happened on latest atmos with both 12.0.0 and 13.1.0, deleting Tesla lets it boot again however.

Not too sure though :p

**Comment by Digital-0xFF** [comment]
Bump

Sys 13.2.1 AMS 1.2.6

While NX-OVLLoader seems to function Tesla Menue leeds to a crash

@WerWolv If you need further info hit me up
